datalad/distribution/publish.py

```diff
@@ -292,14 +353,19 @@ def __call__(
 
         if dataset and since:
```

Comment:
I think this is the issue regarding `since` (Issue #1256 ):
There is no default for `since`. If it is `None`, the commit to consider should be the latest on the remote (last published), right @yarikoptic , @mih ?